pypasta/MailCharacteristics.py

```diff
@@ -327,11 +335,13 @@ def load_maintainers_characteristics(config, characteristics_class, clustering,
                                      ids):
     repo = config.repo
 
-    # We can safely limit to patches only, as only patches will be used for the
-    # maintainers analysis.
-    patches = ids - repo.mbox.invalid
-    tags = {repo.patch_get_version(repo[x]) for x in patches}
-    maintainers_version = load_maintainers(config, tags)
+    maintainers_version = None
+    if characteristics_class.HAS_MAINTAINERS:
+        # We can safely limit to patches only, as only patches will be used for the
+        # maintainers analysis.
+        patches = ids - repo.mbox.invalid
+        tags = {repo.patch_get_version(repo[x]) for x in patches}
+        maintainers_version = load_maintainers(config, tags)
```

Comment:
What about all other projects that do not have HAS_MAINTAINERS yet? They are now broken with this change. Please factor this change out to a preparatory patch, then we get a better overview how we can handle this.